abis_mapping/utils.py

```diff
@@ -0,0 +1,31 @@
+"""Provides utilities for the package"""
+
+
+# Local
+from . import base
+
+
+def get_mapper(template_id: str) -> type[base.ABISMapper]:
+    """Retrieves ABIS Mapper class for the specified template ID.
+
+    Args:
+        template_id (str): Template ID to retrieve the mapper for
+
+    Returns:
+        type[ABISMapper]: ABIS mapper associated with the template ID.
+
+    Raises:
+        KeyError: Raised if the template ID supplied cannot be found
+    """
+    # Retrieve and return the mapper
+    return base.ABISMapper.registry[template_id]
```

Comment:
to return `None` use the `get()` method of dict.
return base.ABISMapper.registry.get(template_id)

Reply:
Resolved in: ff096ef